pysb/importers/bngl.py

```diff
@@ -0,0 +1,319 @@
+from pysb.core import MonomerPattern, ComplexPattern, RuleExpression, \
+    ReactionPattern, ANY, WILD
+from pysb.builder import Builder
+from pysb.bng import BngConsole
+import xml.etree.ElementTree
+import re
+from sympy.parsing.sympy_parser import parse_expr
+import warnings
+
+
+def _ns(tag_string):
+    """
+    Shorthand to apply XML namespace
+    """
+    return tag_string.format('{http://www.sbml.org/sbml/level3}')
+
+
+class BnglImportError(Exception):
+    pass
+
+
+class BnglBuilder(Builder):
```

Comment:
All of the methods and properties introduced in this subclass should really be underscore-prefixed, no? I know we haven't been very consistent with this sort of thing in the past, but with SemVer I am getting a lot more sensitive to exposing things as public by accident. The public interface to this class should probably just be the new constructor and all the public interface from Builder.